Install File Browser Extension Packages (#303)
owncloud/docs-client-desktop: modules/ROOT/pages/installing.adoc

* Note when you start the AppImage after setting it to be executable, AppImageLauncher will open if installed.
--

==== File Browser Extension Packages

> ownCloud supports installing shell / file browser extensions, which means overlay icons and the context menu, via a bash-script-guided installation for Linux-based systems. This script covers main systems and file browsers, but by nature not all possible flavors and combinations. Beside an auto mode, you can also manually define parameters from a list. The following quick command installing the extensions requires `curl` to be available on your system:

Reviewer: We should use one term only. I think so far we've called them shell integration, not shell or file browser extensions.

How about something like this:
```suggestion
ownCloud supports installing shell integration packages, which allow you to add overlay icons and a special context menu to your file browsers, via a bash-script-guided installation for Linux-based systems. This script covers main systems and file browsers, but by nature not all possible flavors and combinations. Beside an auto mode, you can also manually define parameters from a list. The following quick command installing the extensions requires `curl` to be available on your system:
```

Author: That suggestion needed fixing on more locations - done

> ownCloud supports installing shell / file browser extensions, which means overlay icons and the context menu, via a bash script guided installation for Linux based systems. This script covers main systems and file browsers, but by nature not all possible flavours and combinations. Beside an auto mode, you can also manually define parameters from a list. The following quick command installing the extensions requires `curl` to be available on your system:

Reviewer 2: ```suggestion
ownCloud supports installing shell / file browser extensions, which means overlay icons and the context menu, via a bash-script-guided installation for Linux-based systems. This script covers main systems and file browsers, but by nature not all possible flavors and combinations. Beside an auto mode, you can also manually define parameters from a list. The following quick command installing the extensions requires `curl` to be available on your system:
```

[source,bash]
----
curl -s https://raw.githubusercontent.com/owncloud/client-desktop-install-shell-integration/main/install-extensions.sh | bash -s - --auto
----

> If curl ist not available or you are not sure or want to read more, see {install-shell-integration-url}[Install ownCloud Linux Shell Extensions,window=_blank] for more details.

Reviewer 3: ```suggestion
If curl is not available or you are not sure or want to read more, see {install-shell-integration-url}[Install ownCloud Linux Shell Extensions,window=_blank] for more details.
```
